opslevel/resource_opslevel_team.go

```diff
@@ -58,20 +58,20 @@ func resourceTeam() *schema.Resource {
 					Schema: map[string]*schema.Schema{
 						"email": {
 							Type:        schema.TypeString,
-							Description: "The email address or ID of the user to add to a team.",
```

Comment:
"or ID" is not accurate because the read function puts in an email address, so using an ID here is incorrect. Also, it's not practical for users to want to use an ID here.